autobahn/wamp/interfaces.py

```diff
@@ -262,7 +262,10 @@ class ISession(object):
     @abc.abstractmethod
     def onConnect(self):
         """
-        Callback fired when the transport this session will run over has been established.
+        Callback fired when the transport this session will run over has
+        been established.
+
```

Comment:
Okay, I can split it if you want.
I actually could probably just squash to earlier on my other branch, as I had the error-handling stuff but then realized it wasn't handling async-errors properly either. And *then* I didn't have the right args to do tracebacks in the errbacks.

Anyway. I can split it to do "error handling stuff" (without any of the async-errors?) and then a diferent one with the errback API changes + the handling of async-errors? (Then the second one would depend on the first I guess).